index.html

```diff
@@ -26,22 +28,151 @@
+                                    </div></li>
+                                        <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                            <h2 class="card1">Professional Art Printing Data</h2>
+                                            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                            <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                            </div></li>
+                                                <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                                    <h2 class="card1">Professional Art Printing Data</h2>
+                                                    <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                                    <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                        <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                                    </div></li>
+        </ul>
+    </section>
+    <section id="about-section" class="about-section">
+        <div class="about-introduction-content">
+          <h1 class="about-heading">About me</h1>
+          <p>
+            Hey there! I can help you build a product, feature or website. Look
+            through some of my work and experience! If you like what you see and
+            have a project you need coded, don't hesitate to contact me.
+          </p>
+          <button class="my-resume-button" type="button">Get My Resume</button>
+        </div>
+        <div class="background-graphic"></div>
+    <hr class="horizontal-seperator-on-dark" />
+    <div class="about-myself">
+      <div class="language-section">
+        <div class="list-style">
+          <div class="diamond"></div>
+          <h2>Language</h2>
```

Comment:
- [ ] To follow best practices and semantics, I will recommend you to have following structure: 
  - Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.
  - Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.